### ISSUE TITLE
DNS Kaminsky attack

### ISSUE THREAD

**Issue by add1ct3d** [m4s3/DNS-Kaminsky-attack#1]
is this a poc of his method ? and does this work on bind servers?

**Comment by m4s3** [comment]
Yes, that's the implementation of the attack. If you put a place a vm with that dns vulnerable version, you can replicate the attack, just adding the network parameters on the code.

**Comment by add1ct3d** [comment]
this is the old attack ? you didnt answered me

**Comment by m4s3** [comment]
Is there a new attack from kaminsky?

**Comment by m4s3** [comment]
that's the first one.
